lib/modules/platform/gitea/index.spec.ts

```diff
@@ -46,9 +34,8 @@ const GITEA_VERSION = '1.14.0+dev-754-g5d2b7ba63';
 
 describe('modules/platform/gitea/index', () => {
   let gitea: Platform;
-  let helper: jest.Mocked<typeof import('./gitea-helper')>;
   let logger: jest.Mocked<typeof _logger>;
-  let gitvcs: jest.Mocked<typeof _git>;
+  let git: jest.Mocked<typeof _git>;
```

Comment:
why this rename? makes a lot noise and review much harder

Reply:
This is more conventional naming, and there are only two next lines which are using it